static/app/views/settings/organizationAuditLog/auditLogList.tsx

```diff
@@ -231,6 +232,7 @@ function AuditNote({
 }
 
 type Props = {
+  access: boolean;
```

Comment:
you shouldn't need to pass in access as a prop -- `organization.access` contains the access already, and we're fetching it with `useOrganization`